.github/workflows/tests.yml

```diff
@@ -0,0 +1,53 @@
+name: Tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+
+  build_cluster:
+    name: Redis EE (${{ matrix.type }})
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        type:
+          - dmc
+          - oss
+
+    steps:
+
+      - name: Install Docker-compose
+        run: |
+          DOCKER_CONFIG=${DOCKER_CONFIG:-$HOME/.docker}
+          mkdir -p $DOCKER_CONFIG/cli-plugins
+          curl -SL https://github.com/docker/compose/releases/download/v2.23.3/docker-compose-linux-x86_64 -o $DOCKER_CONFIG/cli-plugins/docker-compose
+
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Build cluster
```

Comment:
WDYT: Try the dotnenv loader with the variable change ask?

Reply:
Hmm, does it makes sense to store env variables with values in separate file and use it? What if someone needs to override this values? Values can be pretty much unique for different environments

Reply:
For example image name can be different, especially if you're testing some internal RE images (hello CSC)

Reply:
Done!